hpat/tests/test_series.py

```diff
@@ -2099,6 +2102,62 @@ def test_series_nunique_param1_impl(S, dropna):
                     result_param1 = hpat_func_param1(S, param1)
                     self.assertEqual(result_param1, result_param1_ref)
 
+    def test_series_cumsum(self):
+        def test_impl():
+            series = pd.Series([1.0, np.nan, 9.0, -1.0, 7.0])
+            return series.cumsum()
+
+        pyfunc = test_impl
+        cfunc = hpat.jit(pyfunc)
+        pd.testing.assert_series_equal(pyfunc(), cfunc())
+
+    def test_series_cumsum_unboxing(self):
+        def test_impl(s):
+            return s.cumsum()
+
+        pyfunc = test_impl
+        cfunc = hpat.jit(pyfunc)
+
+        series = pd.Series([1.0, np.nan, 9.0, -1.0, 7.0])
+        pd.testing.assert_series_equal(pyfunc(series), cfunc(series))
+
+    def test_series_cumsum_full(self):
+        def test_impl(s, axis, skipna):
+            return s.cumsum(axis=axis, skipna=skipna)
+
+        pyfunc = test_impl
+        cfunc = hpat.jit(pyfunc)
+
+        series = pd.Series([1.0, np.nan, 9.0, -1.0, 7.0])
```

Comment:
Please see https://github.com/IntelPython/hpat/pull/217#discussion_r335113325

Reply:
Do you mean we need to test implementation on multiple input series, not only a single one?